content/en/docs/reference/kubernetes-api/ref/policies/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+collapsible: true
+draft: false
```

Comment:
nit: Is it important to set the draft field?

Reply:
No, it isn't. I remove draft and collapsible that are not necessary (they inherit from an output for another Hugo theme)